### ISSUE TITLE
Distilled VBench reward model

### ISSUE THREAD

**Issue by wtz222** [mihirp1998/VADER#19]
Thanks for your amazing work!
I am impressed by your approach of distilling VBench into a reward model and fine-tuning with respect to this reward. However, I noticed that neither the code for the distillation process nor the pre-trained reward model is currently available in the repository. ​**Would it be possible for you to share the distillation code** or provide guidance on how to reproduce this step?
Looking forward to your response!
